src/components/IssueFieldForm/CustomField/DateField.tsx

```diff
@@ -6,6 +6,7 @@ import { faCalendarAlt } from "@fortawesome/free-solid-svg-icons";
 
 export const DateField: FC<MappedFieldProps> = ({ id, meta, field, helpers, error }: MappedFieldProps) => (
     <DatePicker
```

Comment:
I'll be looking to remove support for DP UI v1.0 DatePicker component (and variants) in: https://app.shortcut.com/deskpro/story/65388/upgrade-apps-sdk-to-deskpro-ui-2-0